_posts/2024-09-20-release 2.1-preview.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Preview LXQt Release 2.1
+slug: preview-release-2-1
+date: 2024-09-20 21:25
+promoted: true
+categories: blog
+---
+
+Here's a overview of the coming LXQt release 2.1 due in mid November as usual. The main target is the experimental **Wayland session** but also some other improvements will be included.
+
+#### Wayland Session
+
+A new package [lxqt-wayland-session](https://github.com/lxqt/lxqt-wayland-session) will be available to install - without it there won't be seen any changes, following the "modular" philosophy of LXQt: don't bother users with unwanted or unused things. If installed "Session Settings" will look like this:
+
+![Session Settings](../../../../../images/posts/waylandsettings.png)
+
+As Wayland is quite different to X11 some features are not available and some settings are handled by the compositor and not anymore by LXQt, so you may see some messages like that:
```

Comment:
As Wayland is quite different **from** X11,..., so you may see some messages like **this**: